.golangci.yml

```diff
@@ -0,0 +1,307 @@
+# SPDX-FileCopyrightText: 2021 Open Networking Foundation <info@opennetworking.org>
+# Copyright 2019 free5GC.org
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# This file contains all available configuration options
+# with their default values.
+# options for analysis running
+run:
+  # default concurrency is a available CPU number
+  concurrency: 4
+  # timeout for analysis, e.g. 30s, 5m, default is 1m
+  timeout: 1m
+  # exit code when at least one issue was found, default is 1
+  issues-exit-code: 1
+  # include test files or not, default is true
+  tests: true
+  # list of build tags, all linters use it. Default is empty list.
+  build-tags:
+  # which dirs to skip: issues from them won't be reported;
+  # can use regexp here: generated.*, regexp is applied on full path;
+  # default value is empty list, but default dirs are skipped independently
+  # from this option's value (see skip-dirs-use-default).
+  # "/" will be replaced by current OS file path separator to properly work
+  # on Windows.
+  skip-dirs:
+  # default is true. Enables skipping of directories:
+  #   vendor$, third_party$, testdata$, examples$, Godeps$, builtin$
+  skip-dirs-use-default: true
+  # which files to skip: they will be analyzed, but issues from them
+  # won't be reported. Default value is empty list, but there is
+  # no need to include all autogenerated files, we confidently recognize
+  # autogenerated files. If it's not please let us know.
+  # "/" will be replaced by current OS file path separator to properly work
+  # on Windows.
+  skip-files:
+    - "api_.*\\.go$"
+    - "model_.*\\.go$"
+    - "routers.go"
+    - "client.go"
+    - "configuration.go"
+    - "nas.go"
```

Comment:
I think these are not applicable, correct?

Reply:
Removed them.

Reply:
Thanks. but I think you also removed the `*.pb.go`, which I think we should keep due to some files are automatically generated by `protoc-gen-go`